stylus/components/alerts.styl

```diff
@@ -19,7 +19,7 @@ $alerts
         box-sizing        border-box
         box-shadow        0 em(6px) em(18px) 0 rgba(50, 54, 63, .23)
         background-color  emerald
-        padded            .8em, 1em
+        padding            .8em, 1em
```

Comment:
Alignment for the right column is broken

Reply:
Good catch :)